README.md

```diff
@@ -170,12 +174,18 @@ To make more sophisticated assertions about a specific request you can get detai
 
 **For the `fetch` API, we only support string and JSON data!**
 
-### browser.getRequests()
+### browser.getRequests(options)
```

Comment:
Can you copy the arguments here from the `getRequest` function?

Reply:
Yeah, can do. I wasn't sure how verbose the readme API was desired to be, so I elected to be concise and refer back to the `getRequest` description

Reply:
Done!